src/app/shared/components/Navbar/index.tsx

```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image'
+import Link from 'next/link'
+
+interface NavbarProps {
+  title: string,
+}
+
+const NavBar = async (props: NavbarProps) => {
+  const { title } = props
+
+  return (
+    <div className="bg-cyan-700 h-[60px] w-full ">
```

Comment:
i think these hardcoded heights are throwing off our alignment, i think it looks better if we remove the height here and the height and width on line 15